static/js/ducks/profile.js

```diff
@@ -10,6 +10,10 @@ const GENERATE_TOKEN = "skyportal/GENERATE_TOKEN";
 
 const UPDATE_USER_PREFERENCES = "skyportal/UPDATE_USER_PREFERENCES";
 
+const ADD_USER_GCN_PREFERENCES = "skyportal/ADD_USER_GCN_PREFERENCES";
```

Comment:
This should simply be ADD_USER_PREFERENCES and DELETE_USER_GCN_PREFERENCES right?